packages/controller/lib/setup/setupUpload.js

```diff
@@ -568,6 +552,15 @@ function Upload(options) {
         }
 
         if (forceUpload) {
```

Comment:
what was the purpose of the force again? maybe also short jsdoc, if you know it right now